info_extraction/requirements.txt

```diff
@@ -0,0 +1,3 @@
+beautifulsoup4==4.6.0
```

Comment:
Do you need this exact version of bs4? Can a range specification work? Like 4.X is probably fine.

Reply:
I think that was the version of code that was used; it sounds like it would be fine to use 4.X but I know 4.6.0 works